### ISSUE TITLE
community contributed plugin track

### ISSUE THREAD

**Issue by Parul-Kudtarkar** [hammerlab/pileup.js#536]
Hi!

How easy/seamless is it to add community contributed plugin track? Are there any examples/demonstrations available? 

Thanks,
Parul

**Comment by akmorrow13** [comment]
Hi @Parul-Kudtarkar,

We are very open to community contributions! What particular type of track are you interested in adding?

**Comment by Parul-Kudtarkar** [comment]
Hi @akmorrow13, Thanks for confirming! I am thinking of chromatin interactions

**Comment by akmorrow13** [comment]
Yes that would be excellent! @Parul-Kudtarkar what file types store chromatin interactions?

**Comment by Parul-Kudtarkar** [comment]
I am thinking ChIA-PET or Capture Hi-C in .bed format.

**Comment by akmorrow13** [comment]
Ok. The .bed format may be easiest. We currently do not have a DataSource for bed or ChIA-PET. We do, however, support bigbed files. We have been meaning to add bed support for a while but have not yet done it. It should be easy to implement given there is no compression.

**Comment by Parul-Kudtarkar** [comment]
Do you know when the bed format support will be added? Is there anything I can do to assist? Thank you very much!

**Comment by akmorrow13** [comment]
Unfortunately due to loss of man power adding bed support has completely dropped off the table because we already support bigbed files. However, if you can implement the chromatin interaction Visualization Track via a bigbed file, I can then look into adding a bed format to be used by the chromatin interaction visualization. To do this, you can start by copying code for the [Feature Track](https://github.com/hammerlab/pileup.js/blob/master/src/main/viz/FeatureTrack.js), which can read [features](https://github.com/hammerlab/pileup.js/blob/master/src/main/data/feature.js) similar to what you would see in a bed file.

**Comment by Parul-Kudtarkar** [comment]
Excellent! Thank you so much @akmorrow13 I will keep you posted
